### DISCUSSION TITLE
A strange warning in console

### DISCUSSION THREAD

**Discussion by hooke007** [mpvnet-player/mpv.net#420] [General]
Sometimes I opened a file via the menu item (open files)

![Snipaste_2022-05-18_23-59-59](https://user-images.githubusercontent.com/41094733/169169075-5119eb75-6e5b-4856-bed7-09e7a29a8b8b.jpg)

Not sure if it would affect sth. I built from the master and find it.
But would not happen for every file (or not each time. Really weird).

**Comment by stax76** [comment]
wm4 and I were involved in this:

https://github.com/mpv-player/mpv/issues/7501

Please note that I have terrible memory.

**Comment by stax76** [comment]
Thing is annoying IMO. For development, I'm in the terminal quite often.
